src/utils/eventAccessibilityLinter.ts

```
@@ -0,0 +1,127 @@
+
+    const fg = styles['color'];
+    const bg = styles['background-color'] || styles['background']; // very simplified
+
+    if (fg && bg) {
+      const fgColor = parseColor(fg);
+      const bgColor = parseColor(bg);
+
+      if (fgColor && bgColor) {
+        const ratio = calculateContrastRatio(fgColor, bgColor);
+        if (ratio < MIN_CONTRAST_RATIO) {
+          errors.push(`Contrast ratio between text (${fg}) and background (${bg}) is too low (${ratio.toFixed(2)}:1). WCAG AAA requires at least ${MIN_CONTRAST_RATIO}:1.`);
```

ESLint

[error] Replace ``Contrast·ratio·between·text·(${fg})·and·background·(${bg})·is·too·low·(${ratio.toFixed(2)}:1).·WCAG·AAA·requires·at·least·${MIN_CONTRAST_RATIO}:1.`` with `⏎············`Contrast·ratio·between·text·(${fg})·and·background·(${bg})·is·too·low·(${ratio.toFixed(2)}:1).·WCAG·AAA·requires·at·least·${MIN_CONTRAST_RATIO}:1.`,⏎··········`

(prettier/prettier)